src/bluetooth.py

```diff
@@ -0,0 +1,30 @@
+import bluetooth
+import pexpect
+import time
+
+def connect(targetName):
+    targetAddress = None
+
+    while targetAddress == None:
+        nearby_devices = bluetooth.discover_devices()
+        print(len(nearby_devices))
+        for i in nearby_devices:
+            print(bluetooth.lookup_name(i))
+            if targetName == bluetooth.lookup_name(i):
+                targetAddress = i
+
+        time.sleep(1)
```

Comment:
Noted, I can take that out. I just slept for testing.

Reply:
I don't think we need to scan anyway. If we're paired to a device, we should just be able to connect. We don't need to put the Pi in discovery mode again because that searches for devices that we haven't paired with. We definitely don't need to be printing and that should be taken out before pushing. It's extremely useful for development, but it's not helpful for the actual software and will slow it down a lot. Printing is a blocking operation so taking it out should make this faster.